core/packet.c

```diff
@@ -198,6 +198,11 @@ static uint8_t handle_request(lwm2m_context_t * contextP,
     case LWM2M_REQUEST_TYPE_REGISTRATION:
         result = registration_handleRequest(contextP, &uri, fromSessionH, message, response);
         break;
+#ifndef LWM2M_VERSION_1_0
```

Comment:
It might be useful to add a check for `LWM2M_SERVER_MODE` in addition to `LWM2M_VERSION_1_0`.

Reply:
Is already in a #ifdef LWM2M_SERVER_MODE (line 197)

Reply:
Yes, my bad.